modules/checklist/steps-manager.php

```diff
@@ -71,13 +77,7 @@ public function mark_step_as_completed( string $step_id ) : void {
 	 * @return void
 	 */
 	public function unmark_step_as_completed( string $step_id ) : void {
```

Comment:
I wonder, if instead of having two functions `unmark_step_as_completed` and `mark_step_as_completed ` should we have one function called `set_step_state` and pass the desire state to it?
E.g,
set_step_state( string $step_id, bool true );
set_step_state( string $step_id, bool false );

WDYT?

Reply:
I was actually wondering the same thing, I think that having a `->mark_as_completed` makes life a little bit easier but if you feel this is not the case I'll change it

Reply:
I am cool with it.